examples/webhooks/webhooks_test.go

```diff
@@ -0,0 +1,88 @@
+package webhooks
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+
+	"github.com/moovfinancial/moov-go/pkg/mhooks"
+	"github.com/moovfinancial/moov-go/pkg/moov"
+)
+
+func TestNewEvent(t *testing.T) {
```

Comment:
I will say if you're going to table test every event this test is going to get extremely long and be unreadable. I would think about how you're going to organize this so it makes sense.

Reply:
Given this is in the `examples` directory, I'm not going to add more test cases for each event type; main goal is to showcase how you can create a handler to handle more than 1 event type using a `switch` statement

Reply:
Since this is examples we should remove the table tests all together and set it up so it’s readable and closely match the intended use.

Reply:
I'll update this in a new PR -- I want to see what this looks like with the signature verification changes.